docs/architecture/adr-108-e2e-abci++.md

```diff
@@ -0,0 +1,305 @@
+# ADR 108: E2E tests for CometBFT's behaviour in respect to ABCI++.
+
+## Changelog
+- 2023-08-08: Initial version (@nenadmilosevic95)
+
+
+## Context
+
+ABCI++ defines the interface between the application and CometBFT. A part of the specification is the ABCI++ grammar that describes the sequences of calls that the application can expect from CometBFT.
```

Comment:
```suggestion
ABCI 1.0 defines the interface between the application and CometBFT. A part of the specification is the ABCI++ grammar that describes the sequences of calls that the application can expect from CometBFT.
```
Please, change ABCI++ to ABCI 1.0 all over. (ABCI 1.0 -> `PrepProp` and `ProcProp` but no vote extensions).
Whereas ABCI++ tends to be identifed with the whole lot (a.k.a. ABCI 2.0)